redaxo/src/core/lib/console/db/dump_schema.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ *
+ * @author gharlan
+ *
+ * @internal
+ */
+class rex_command_db_dump_schema extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Dumps the schema of db tables as php code')
+            ->addArgument('table', InputArgument::REQUIRED, 'Database table')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $table = rex_sql_table::get($input->getArgument('table'));
+
+        $generator = new rex_sql_schema_dumper();
+
+        $output->write($generator->dump($table));
```

Comment:
Success/error msg?

Reply:
Die Ausgabe ist ja der PHP-Code. Da würde ich keine weiteren Messages ausgeben, damit man auch direkt in Dateien schieben kann..
`rex db:dump-schema > file.php`

Reply:
Ggf auf stderr (wie es cli tools normalerweise tun)

Reply:
S. A. https://stackoverflow.com/a/1188209/1597388